spec/unit/knife/bootstrap_spec.rb

```diff
@@ -593,6 +593,20 @@ def certificates
         knife.run
       end
     end
+
+    context "when the validation_key is nil" do
+      before do
+        # this tests runs the old code path where we have a validation key, so we need to pass that check for some plugins
+        Chef::Config[:validation_key] = nil
+      end
+
+      it "creates the client (and possibly adds chef-vault items)" do
```

Comment:
This test does something different than the test in the previous context, so the name needs to be different. From what I gather, this is "creates the client and does not run client_builder or the chef_vault_handler"